src/preload.py

```diff
@@ -288,6 +290,8 @@ def _resize_partition_on_disk_image(self, additional_bytes):
         # partition (some devices rely on the bootloader being there, like the
         # Variscite DART-6UL)
         if self.partition_table.label == "dos":
```

Comment:
Do GPT-partitioned disks also need to have this bootstrap-code area copied? (I googled and found https://askubuntu.com/questions/657179/where-is-grub-installed-on-a-gpt-disk )

Reply:
I don't think we have any image that uses the "BIOS-mode booting from GPT" but as there is a protective MBR in the beginning of a GPT partitionned disk I think we can safely copy the first 446 bytes in all cases.
Sounds good @lurch ?

Reply:
code updated to copy the first 446 bytes in all cases